docs/guides/registries/azure-acr.md

```diff
@@ -63,9 +63,9 @@ Once you are logged in, and have the optional credential helper installed, then
 ```
 ❯ ../earthly/earthly --push --no-cache +build
            buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
-         alpine:3.15 | --> Load metadata linux/amd64
-               +base | --> FROM alpine:3.15
-               +base | [██████████] resolve docker.io/library/alpine:3.15@sha256:0bd0e9e03a022c3b0226667621da84fc9bf562a9056130424b5bfbd8bcb0397f ... 100%
+         alpine:3.18 | --> Load metadata linux/amd64
+               +base | --> FROM alpine:3.18
+               +base | [██████████] resolve docker.io/library/alpine:3.18@sha256:0bd0e9e03a022c3b0226667621da84fc9bf562a9056130424b5bfbd8bcb0397f ... 100%
```

Comment:
I'm guessing that sha here will be different 😆 
Here, I found it :)
```suggestion
               +base | [██████████] resolve docker.io/library/alpine:3.18@sha256:c5c5fda71656f28e49ac9c5416b3643eaa6a108a8093151d6d1afc9463be8e33 ... 100%
```
https://hub.docker.com/layers/library/alpine/3.18/images/sha256-c5c5fda71656f28e49ac9c5416b3643eaa6a108a8093151d6d1afc9463be8e33?context=explore

Reply:
I don't think this is necessary; plus these shas can change whenever the patch version changes.